prometheus/README.md

```diff
@@ -0,0 +1,29 @@
+# Prometheus Event Source for Knative Eventing
+
+The Prometheus Event Source source enables Knative Eventing integration with the
+[Prometheus Monitoring System](https://prometheus.io/).
+
+At the moment the implementation supports only simple PromQL queries. The queries are
+run against the Prometheus server every 5 seconds and the results are sent
+to the configured sink as CloudEvents.
+
+## Deployment steps
+
+1. Setup [Knative Eventing](../DEVELOPMENT.md)
+1. Create a PrometheusSource CRD, specifying the URL for the Prometheus server,
+   the PromQL query to run and the sink to send CloudEvents to:
+
+   ``` yaml
+   apiVersion: sources.eventing.knative.dev/v1alpha1
+   kind: PrometheusSource
+   metadata:
+     name: prometheus-source
+     namespace: knative-sources
```

Comment:
let's remove this `ns`, so (like other samples), it goes to `default` .

Reply:
Fixed.